openbb_platform/core/openbb_core/build.py

```diff
@@ -0,0 +1,39 @@
+"""Script to build the OpenBB platform static assets."""
+
+# flake8: noqa: S603
+# pylint: disable=import-outside-toplevel
+
+import subprocess
+import sys
+
+
+def main():
+    """Build the OpenBB platform static assets."""
+    message = "\nOpenBB build script not found, installing from PyPI...\n"
```

Comment:
You can move this down where it is used. I'd also explore not printing and logging instead or doing an echo.

Reply:
Can do as an echo.